tools/build_framework.sh

```diff
@@ -33,7 +33,6 @@ shift
 echo "Building $FRAMEWORK_NAME in $FRAMEWORK_DIR:"
 
 # optional args:
-cli_only=
 custom_artifacts=
 publish_method="no"
 while [ $# -gt 0 ]; do
```

Comment:
Out of scope of this PR, but we could consider settling on a single way to handle command-line arguments. We use `getopts` elsewhere.

Reply:
Ended up doing this as part of `build_package.sh`.